Src/Sarif.PatternMatcher/Sarif.PatternMatcher.csproj

```diff
@@ -10,18 +10,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
>12.0.3 [](http://example.com/codeflow?start=57&length=6)

Reading the description, I don't see an explanation of the reason for this change. Is this to prevent a conflict with the versions of RE2.Managed and Strings.Interop? How does this affect package flattening?

Reply:
Observed an issue with Hulon when compiling in vs/cmd which shows many warnings.

With that, I want to fix this and make things cleaner :)

Reply:
Actually, I wrote it in the description: `It also changes the Sarif.PatternMatcher projects to depend on RE2.Managed/Strings.Interop package to prevent some warnings when building in the cmd/vs.`

Reply:
So, the part that is missing from the description is that this is to cleanup build warnings by aligning newtonsoft.json package between spam and internal references. Cool.